Code/Framework/AzNetworking/AzNetworking/Utilities/QuantizedValues.inl

```diff
@@ -300,12 +300,12 @@ namespace AzNetworking
         static inline void DecodeQuantizedValues(SelfType& quantizedValues)
         {
             constexpr double maximumInt     = static_cast<double>(MaxSerializedIntValue);
-            constexpr double minimumFloat   = static_cast<double>(MIN_VALUE);
+            constexpr float minimumFloat   =  static_cast<float>(MIN_VALUE);
```

Comment:
Since MIN_VALUE is an `int32_t` casting it to a float causes precision loss as all of the values can't fit all 32-bit int values can't fit within a float. It can only represent integer values between -2^24 to 2^24

`minimumFloat` begin a double allows it to represent integers values up between -2^53 to 2^53 exactly.

Reference: https://stackoverflow.com/a/12445693

Therefore I think a second static cast should occur.

```
quantizedValues.m_quantizedValues[i] = static_cast<float>(minimumFloat + static_cast<float>(quantized * convertToFloat));
```

Reply:
Will give this a try

Reply:
Well at least we can defer that precision lost until after the last calculation.